rules/gcp_audit_rules/gcp_destructive_queries.py

```diff
@@ -0,0 +1,79 @@
+from panther_base_helpers import deep_get
+
+DESTRUCTIVE_STATEMENTS = ("UPDATE", "DELETE", "DROP_TABLE", "ALTER_TABLE", "TRUNCATE_TABLE")
```

Comment:
it seems weird to me to do  `item in tuple`. Could this be retyped to either list or set?